ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -542,4 +551,30 @@ void testSameHttpClientInstance() throws DataClientException, DataServiceExcepti
 
         Mockito.verify(httpClientSpy, atLeast(2)).execute(any());
     }
+
+    @Test
+    void testNoRedirects() throws Exception {
+        List<Integer> redirectCodes = Arrays.asList(301, 302, 303, 307, 308);
+        CloudInfo.manuallyAddToCache("https://fake.kusto.windows.net", CloudInfo.DEFAULT_CLOUD);
```

Comment:
Do we confirm both CloudInfo and the client itself prevent redirects?
Do they share the same http instance